frontend/src/pages/LogsPage/useGetLogs.ts

```diff
@@ -29,111 +29,97 @@ export const useGetLogs = ({
 	startDate: Date
 	endDate: Date
 }) => {
-	// The backend can only tell us page info about a single page.
-	// It has no idea what pages have already been loaded.
-	//
-	// For example: say we make the initial request for 100 logs and hasNextPage=true and hasPreviousPage=false
-	// That means that we should not make any requests when going backwards.
-	//
-	// If the user scrolls forward to get the next 100 logs, the server will say that hasPreviousPage is `true` since we're on page 2.
-	// Hence, we track the initial information (where "window" is effectively multiple pages) to ensure we aren't making requests unnecessarily.
-	const [windowInfo, setWindowInfo] = useState<PageInfo>({
-		hasNextPage: false,
-		hasPreviousPage: false,
-		startCursor: '',
-		endCursor: '',
-	})
+	const [hasNextPage, setHasNextPage] = useState(true)
+	const [hasPreviousPage, setHasPreviousPage] = useState(false)
```

Comment:
looks like the changes have fixed one bug where we would have incorrect results on follow-up queries, but we still have an issue where a query has no pages even when it should: https://www.loom.com/share/f5424be4731649b49fc640ee9752aadd

i think the culprit is that these two states need to be reset when the `serverQuery` or `startDate` and `endDate` change, as those values should retrigger the primary page query. currently, we set `hasNextPage` to a default `true` value on the intial load of this hook, but maybe these two values should be undefined and updated on `data` changing?